lib/Dialect/FIRRTL/FIRRTLOps.cpp

```diff
@@ -383,13 +383,14 @@ static LogicalResult verifyCircuitOp(CircuitOp circuit) {
         else
           leafInnerSyms.insert(op, modName);
       });
-    }
-    if (auto mod = dyn_cast<FModuleLike>(op))
       leafInnerSyms.insert(mod);
+    }
     // Verify external modules.
-    if (auto extModule = dyn_cast<FExtModuleOp>(op))
+    if (auto extModule = dyn_cast<FExtModuleOp>(op)) {
+      leafInnerSyms.insert(extModule);
```

Comment:
I think you need to copy this above  for the `FModuleOp` case too

Reply:
Yes, added that, line 386.